ground/src/main/java/com/google/android/ground/ui/tos/TermsOfServiceViewModel.kt

```diff
@@ -29,7 +29,7 @@ constructor(
   private val navigator: Navigator,
   private val termsOfServiceRepository: TermsOfServiceRepository
 ) : AbstractViewModel() {
-  // TODO(Shobhit): Convert to MutableLiveData.
+  // TODO(https://github.com/google/ground-android/issues/1478): Convert to MutableLiveData.
```

Comment:
I've been using the convention `TODO(#issue no)` since the issue link isn't clickable in code anyway, but I don't feel strongly about this.

Reply:
Reverted for consistency.